docs/2_QUICK_START_GUIDE.md

```diff
@@ -1,13 +1,229 @@
+    az account show # A command for validating the azure cli is correctly configured
+    az configure --defaults group='' # Use empty strings to clear default values for any sticky az properties like "group"
+    ```
+
+### **Step 4:** Initialize a Terraform Remote Environment
+
+A Terraform environment is primarily any json file generated by the `terraform init` and/or `terraform workspace new` commands. Initializing a Terraform *remote* environment (as opposed to a local one) effectively means to create an Azure blob storage json file within your existing Azure blob storage account and container listed in the [prerequisites](#23-prerequisites).
+
+* Navigate to the az-hello-world directory (i.e. ./infra/templates/az-hello-world) and execute the following commands to set up your remote terraform environment.
+
+    ```bash
+    # This configures terraform to leverage a remote backend that will help you and your
+    # team keep consistent state
+    terraform init -backend-config "storage_account_name=${TF_VAR_remote_state_account}" -backend-config "container_name=${TF_VAR_remote_state_container}"
+
+    # This command configures terraform to use a workspace unique to you. This allows you to work
+    # without stepping over your teammate's deployments
+    terraform workspace new "az-hw-$USER" || terraform workspace select "az-hw-$USER"
+    ```
+
+    ```bash
+    $ tree infra
+    ├───.env.template
+    ├───.env
+    ├───modules
+    └───templates
+        ├───az-hello-world
+        │   └───.terraform ## NEW LOCAL DIRECTORY GENERATED FROM INITIALIZING REMOTE STATE
+        └───backend-state-setup
+    ```
+
+### **Step 5:** Deploy Cobalt's [_Azure Hello World CIT_](../infra/templates/az-hello-world/README.md)
+
+This is the step that demonstrates what the [_Azure Hello World CIT_](../infra/templates/az-hello-world/README.md) has to offer by deploying and creating actual Azure cloud resources. This infrastructure template creates an [ARM Resource Group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview), an [App Service Plan](https://docs.microsoft.com/en-us/azure/app-service/overview-hosting-plans) and [App Service](https://docs.microsoft.com/en-us/azure/app-service/) running a public docker container. The previous step of initializing a Terraform environment from within the Azure Hello World CIT directory makes deployment possible.
+
+1. From the az-hello-world directory, execute the following commands to orchestrate a deployment.
+
+    ```bash
+    # See what terraform will try to deploy without actually deploying
+    terraform plan
+
+    # Execute a deployment
+    terraform apply
+    ```
+
+1. Take note of the name of the resource group output from Terraform apply.
+
+* **Azure Hello World CIT [Main.tf](https://github.com/microsoft/cobalt/blob/master/infra/templates/az-hello-world/main.tf) File**
```

Comment:
Can we omit the copy of the file? When the impl changes, it will be difficult to keep this up to date.

Also we are using a non-local ref to the template